packages/matrix/scripts/register-test-user.ts

```diff
@@ -1,17 +1,17 @@
 import * as childProcess from 'child_process';
 
 import { loginUser } from '../docker/synapse';
-
 export const adminUsername = 'admin';
 export const adminPassword = 'password';
 
-let username = process.env.USERNAME || adminUsername;
-let password = process.env.PASSWORD || adminPassword;
```

Comment:
USERNAME and PASSWORD are very generically used env variables. Might trip people up